source/includes/pipeline_config/_023-tab.md.erb

```diff
@@ -0,0 +1,16 @@
+<%
+json ='{
+  "tabs": [
+    {
+      "name": "cobertura",
+      "path": "target/site/cobertura/*.xml"
+    }
+  ]
+}'
+%>
+<%=
+describe_sub_object 'The tab object', json: json do
+  name          'String',       'The name of the tab which will appear in the Job detail page.'
+  path          'String',       'The path of the artifact that will be render in the tab.'
```

Comment:
Technically this is the relative path of a file in the server artefact destination directory of the job.

Not sure how to phrase it well.